src/ocm/components/clusterConfiguration/platformIntegration/PlatformIntegration.tsx

```diff
@@ -43,17 +43,32 @@ const PlatformIntegrationLabel = ({
   );
 };
 
-const PlatformIntegration = ({ clusterId }: { clusterId: Cluster['id'] }) => {
+const PlatformIntegration = ({
+  clusterId,
+  openshiftVersion,
+}: {
+  clusterId: Cluster['id'];
+  openshiftVersion: Cluster['openshiftVersion'];
+}) => {
   const { isPlatformIntegrationSupported, supportedPlatformIntegration } =
     useClusterSupportedPlatforms(clusterId);
 
+  const featureSupportLevels = useFeatureSupportLevel();
+  const isNutanixSupported = featureSupportLevels.isFeatureSupported(
```

Comment:
```suggestion
  const isNutanixFeatureSupported = featureSupportLevels.isFeatureSupported(
```

I suggest the change to have more difference between `isPlatformIntegrationSupported` and `isNutanixSupported`, otherwise they can be confused.

Reply:
Done :+1: